Marlin/language.h

```diff
@@ -31,8 +31,8 @@
 // kana     Japanese
 // kana_utf Japanese
 
+// fallback if no language is set, don't change
```

Comment:
@thinkyhead What do you think? I think there should be at least a warning like
this.